### ISSUE TITLE
Remove Buffer from being included in web by default

### ISSUE THREAD

**Issue by eliperelman** [mozilla-neutrino/neutrino-dev#196]
Adding `Buffer` to the Node.js configuration for the Web preset includes 50KB of polyfills, for a feature that mostly likely won't even be used. We should set this value to `false`, and make this feature opt-in.

**Comment by barraponto** [comment]
I wonder why we set every [default value for the node configuration](https://webpack.js.org/configuration/node/) in our [web-preset](https://github.com/mozilla-neutrino/neutrino-dev/blob/master/packages/neutrino-preset-web/index.js#L123-L133). Can't we just set the overrides?

**Comment by barraponto** [comment]
Btw, the [documentation](https://webpack.js.org/configuration/node/#other-node-core-libraries) says:
> By default, Webpack will polyfill each library if there is a known polyfill or do nothing if there is not one.

Does it *really* mean that we're polyfilling every node library by default?

**Comment by eliperelman** [comment]
Closed via #201.

**Comment by eliperelman** [comment]
So the defaults are:

```js
node: {
  console: false,
  global: true,
  process: true,
  __filename: "mock",
  __dirname: "mock",
  Buffer: true,
  setImmediate: true
}
```

And yes, we just re-set the defaults. I can't remember why this is, but it's probably no longer needed. Only things marked `true` here would be polyfilled by default.
